### PR TITLE
Updates Dependencies & Update Build - Not Complete Yet (v4-Dev)

### PR DESCRIPTION
**Pull request by CDAGaming**
Did not know whether these dependencies are meant to stay outdated, so I made this anyways. 
New Build files for this will come along shortly as well.

### CODE REVIEW

**Comment by twbs-rorschach** [comment]
Hi @CDAGaming!

Thanks for wanting to contribute to Bootstrap by sending this pull request!
Unfortunately, your pull request seems to have some problems:
* [Changes must be made to the original Sass or Less source code file(s), not just the compiled CSS file(s).](https://github.com/twbs/rorschach/blob/master/docs/css.md)

You'll need to **fix these mistakes** and revise your pull request before we can proceed further.
Once you've fixed these problems, you can either ask the maintainers to re-open this pull request, or you can create a new pull request.
Thanks!

(*Please note that this is a [fully automated](https://github.com/twbs/rorschach) comment.*)

**Comment by twbs-rorschach** [comment]
Hi @CDAGaming!

Thanks for wanting to contribute to Bootstrap by sending this pull request!
Unfortunately, your pull request seems to have some problems:
* [Changes must be made to the original Sass or Less source code file(s), not just the compiled CSS file(s).](https://github.com/twbs/rorschach/blob/master/docs/css.md)

You'll need to **fix these mistakes** and revise your pull request before we can proceed further.
Once you've fixed these problems, you can either ask the maintainers to re-open this pull request, or you can create a new pull request.
Thanks!

(*Please note that this is a [fully automated](https://github.com/twbs/rorschach) comment.*)

File: package.json [hunk 1]
```diff
@@ -70,13 +70,13 @@
   },
   "license": "MIT",
   "dependencies": {
-    "jquery": ">=1.9.1",
+    "jquery": ">=3.2.1",
```
Comment (wolfy1339) [inline package.json]: This should stay as it is. Bootstrap v4 works with versions 1.9.1 of jQuery and up

**Review by wolfy1339 — COMMENTED** [review]
Please remove all charges to dist files

**Comment by CDAGaming** [comment]
@wolfy1339 Ok, will keep the other updates, put jQuery back to origin version, remove changes to the dist files, and re PR shortly

**Pull request closed without merging**